introspection-engine/introspection-engine-tests/tests/tables/mod.rs

```diff
@@ -4,15 +4,21 @@ use introspection_engine_tests::{assert_eq_datamodels, test_api::*};
 use quaint::prelude::Queryable;
 use test_macros::test_each_connector_mssql as test_each_connector;
 
-#[test_each_connector]
+#[test_each_connector(tags("postgres", "mysql", "sqlite"))]
 async fn a_simple_table_with_gql_types(api: &TestApi) -> crate::TestResult {
+    let sql_family = api.sql_family();
+
     api.barrel()
         .execute_with_schema(
             |migration| {
-                migration.create_table("Blog", |t| {
+                migration.create_table("Blog", move |t| {
                     t.add_column("bool", types::boolean());
                     t.add_column("float", types::float());
-                    t.add_column("date", types::datetime());
+                    if sql_family.is_postgres() {
+                        t.inject_custom("date Timestamp Not Null");
```

Comment:
```
t.add_column("date", types::timestamp().nullable(false))
```

Reply:
If missing that function, you can add it to this branch: https://github.com/prisma/barrel/tree/mssql-support

Reply:
There is no timestamp type, but time works as well for the purposes of this test.